index.js

```diff
@@ -343,6 +358,8 @@ module.exports = {
         var relationships = {};
         _.each(item.extras || {}, function(val, key) {
           var newKey = 'a205file' + key;
+          Object.assign(val, val.crop || {});
```

Comment:
I don't understand the purpose of adding `val.crop` and then removing it on the next line.